src/Administration/Resources/app/administration/src/module/sw-cms/elements/image-gallery/config/index.js

```diff
@@ -22,7 +22,7 @@ export default {
         return {
             mediaModalIsOpen: false,
             initialFolderId: null,
-            enitiy: this.element,
+            entity: this.element,
```

Comment:
This is a breaking change. The typo property needs to stay and you need to add a comment above
/* @deprecated tag:v6.6.0 - Will be removed use entity instead. */

Reply:
Updated it and change changelog to "Deprecated"